paint/PAINT_ColourPicker.h

```diff
@@ -20,21 +21,29 @@ namespace paint
 		ColourPicker& operator=(const ColourPicker & that) = default;
 		ColourPicker& operator=(ColourPicker && that) = default;
 
+		void initialiseColourDisplays();
+		void initialiseColourValueBoxes();
+		void initialiseColourSliders();
+		
 		std::shared_ptr<win::ColourValueTextbox> getRedValueBox() const { return redValueBox_;  }
 		std::shared_ptr<win::ColourValueTextbox> getGreenValueBox() const { return greenValueBox_;  }
 		std::shared_ptr<win::ColourValueTextbox> getBlueValueBox() const { return blueValueBox_;  }
 		std::shared_ptr<win::ColourValueTextbox> getAlphaValueBox() const { return alphaValueBox_;  }
 		
 		void setDrawColourFromActive() const;
 		void setPrimarySecondaryFromActiveDraw() const;
+		void updateColourDisplaysFromDrawWindow() const;
+		void updateColourDisplays() const;
 		void updateColourValueBoxes() const;
-		void swapActiveColour();
-		void updateColourDisplaysFromDrawWindow();
+		void updateColourSliders() const;
+		void swapActiveColour() const;
 		void swapDisplaysSwitch() { swappedDisplays_= !swappedDisplays_; }
```

Comment:
This is quite a big API. Do they all need to be public? The mantra for class design is that the public API should be only as big as it needs to be for the class to perform its single role. Decoupling from the draw window (see previous comments) would help here maybe.
**COULD**